app/views/friendships/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<section class="users-section">
+<h3>Pending Requests</h3>
+  <ul class="users-list">
+    <% @pending_friends.each do |user| %>
+      <li>
+        Name: <%= user.name %>
+        <span class="profile-link">
+
+          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
+          
+          <% if current_user != User.find(user.id)%>
+            <% if show_add_friend(user) %>
+              <%= link_to 'Invite to friendship', {:controller => 'friendships', :action => 'create', :id => user.id}, method: :post, class: 'profile-link' %>
+            <% end %>
+
+            <% if show_already_friend(user) %>
+              <span> | you are friends</span>
+            <% end %>
+          <% end %>
+        </span>
+      </li> 
+    <% end %>
```

Comment:
Extract the logic in friendships view into helper methods.

Reply:
Issue solved